mailbox/plugin/quota-mailing/src/main/java/org/apache/james/mailbox/quota/model/QuotaThreshold.java

```diff
@@ -0,0 +1,99 @@
+/****************************************************************
+ * Licensed to the Apache Software Foundation (ASF) under one   *
+ * or more contributor license agreements.  See the NOTICE file *
+ * distributed with this work for additional information        *
+ * regarding copyright ownership.  The ASF licenses this file   *
+ * to you under the Apache License, Version 2.0 (the            *
+ * "License"); you may not use this file except in compliance   *
+ * with the License.  You may obtain a copy of the License at   *
+ *                                                              *
+ *   http://www.apache.org/licenses/LICENSE-2.0                 *
+ *                                                              *
+ * Unless required by applicable law or agreed to in writing,   *
+ * software distributed under the License is distributed on an  *
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY       *
+ * KIND, either express or implied.  See the License for the    *
+ * specific language governing permissions and limitations      *
+ * under the License.                                           *
+ ****************************************************************/
+
+package org.apache.james.mailbox.quota.model;
+
+import java.util.Objects;
+import java.util.Optional;
+
+import org.apache.james.mailbox.model.Quota;
+
+import com.google.common.base.MoreObjects;
+import com.google.common.base.Preconditions;
+
+public class QuotaThreshold implements Comparable<QuotaThreshold> {
+
+    public static final QuotaThreshold ZERO = new QuotaThreshold(0.);
```

Comment:
I didn't knew this `0.` notation, I usually use `0d` that looks more explicit. Maybe the team has an opinion about that ?

Reply:
You mean 0f? Anyway I don't care, 0. is pretty common (at least in Caml world :))

Reply:
Oh, and `0.` is `0f` not `0d` but you expected a double (: 
I'm ok with keeping it that way